testData/solutions/win32.json5

```diff
@@ -2617,6 +2660,44 @@
                             "default": 0
                         }
                     },
+                    "OutputModes.ACCESS_KEY": {
+                        "schema": {
+                            "title": "Access Key",
+                            "description": "Which items to speak with the access key???",
+                            "type": "string",
+                            "default": "1|1|1|Access Key",
+                            "enum": [
+                                "0|0|0|Access Key",
+                                "1|1|1|Access Key",
+                                "2|2|2|Access Key",
+                                "3|3|3|Access Key"
+                            ],
+                            "enumLabels": [
+                                "Off",
+                                "Speak all",
+                                "Speak menus only",
+                                "Speak dialogs only"
+                            ]
+                        }
+                    },
+                    "OutputModes.TUTOR": {
+                        "schema": {
+                            "title": "Tutor Messages",
+                            "description": "Which items to speak with the access key???",
+                            "type": "string",
+                            "default": "1|1|1|Tutor",
+                            "enum": [
+                                "0|0|0|Tutor",
```

Comment:
Same question here, is the literal value `0` here or the string you've put here?

Reply:
Yup, same as above, this is the actual odd string value that shows up in the ini formatted settings.